packages/actor-dereference-file/lib/ActorDereferenceFile.ts

```diff
@@ -22,13 +22,17 @@ export class ActorDereferenceFile extends ActorDereference {
   }
 
   public async run({ url }: IActionDereference): Promise<IActorDereferenceOutput> {
+    // E.g., sparql@http://example.org/
```

Comment:
If I remember correctly, these tagged URLs will never occur at this stage, they should already be stripped away at an earlier stage.

So I think we only have to check for the presence of a scheme, so just checking for `.indexOf(':') > 0` should be sufficient.

Reply:
Good point, indeed tags are stripped out at this point.
As much as I'd like to not have a regex in favor of something simpler, it seems if want to be sure that we don't have false positive matches, we better make it as strict to a URI shape check as we can..
E.g., `C:\file.ttl` would pass if we are to check for `:` presence solely
Perhaps we can't escape having a regex here..